layersvt/VkLayer_device_simulation.def

```diff
@@ -26,4 +26,5 @@ vkGetInstanceProcAddr
 vkCreateInstance
 vkEnumerateInstanceLayerProperties
 vkEnumerateInstanceExtensionProperties
-vkNegotiateLoaderLayerInterfaceVersion
+vkEnumeratePhysicalDevices
+vkNegotiateLoaderLayerInterfaceVersion
```

Comment:
may not have been easily visible, but GitHub complains that there's no newline at the end of this file.